setup_new.py

```diff
@@ -123,7 +125,7 @@ def pip_install(package_name):
 else:
     if os.path.isfile(venv_exec):
         #print("Activating virtual environment")
-        cont = input("Activate virtual environment located at %s? [y/n]:")
+        cont = input("Activate virtual environment located at %s? [y/n]:"%venv_exec)
```

Comment:
Maybe we should gradually move towards the [new python string formatting](https://docs.python.org/3/library/string.html#format-specification-mini-language)

The above line could then be written 

`f"Activate virtual environment located at {venv_exec}? [y/n]:"`